### DISCUSSION TITLE
App name poll (second round)

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#52] [Polls]
The initial poll for the app name provided inconclusive results (there was no clear majority). This marks the second round, featuring new names suggested by the community. The outcome of this poll will be taken into account only if it achieves a minimum of 15% of the votes from the [initial poll.](https://github.com/orgs/FossifyOrg/discussions/23)

As the apps are not yet ready for release, a third and final poll will be conducted if the top-voted name in this round does not secure at least 65% of the total votes.

**Comment by nehemiagurl** [comment]
still a shame one can't vote on multiple options :( but I guess that's life with GitHub polls.

**Reply by bmslgpn** [reply]
I agree, I suggested to use https://app.mieuxvoter.fr/en. I like it.

**Comment by DiagonalArg** [comment]
I think I would have liked "Fossy" the best, but I'll go for Fossify and maybe change my vote to Foss if it looks like things are going that way.

(Ah, maybe we can't change our vote.  No matter!)

(Oh, yes we can, see the next post!)

**Reply by naveensingh** [reply]
You can change your vote! Here's how:
- Click on **Hide results** at the bottom.
- Choose the name you want.
- Click **Vote** again.

**Reply by nehemiagurl** [reply]
Fossy us great! +1 to that

**Comment by erebion** [comment]
"fossify" sounds like a verb, therefore "fossify gallery" sounds like "make a gallery FOSS", which in turn does not seem to be a gallery app but sounds like a project that makes software FOSS. Which is what this org does, not the apps.

"fossify gallery" seems more like it could be a gallery of projects that were open sourced.

**Reply by erebion** [reply]
Also, names should be simple while conveying what it is all about, here it is mainly about being a simple gallery app which is FOSS, so "FOSS Gallery" works well.

Simple, yet it conveys the essence.

**Reply by ikanakova** [reply]
But there are already apps that have FOSS in their name, such as FOSS Browser, FOSS Weather, etc. Fossify (and probably Fossy) would be unique.

**Reply by LearningAsIGo71** [reply]
Besides, naming an app FOSS this or FOSS that, is super generic.

**Reply by erebion** [reply]
Exactly, that's what makes it great, just simple FOSS apps.

**Comment by ciampix** [comment]
I always suggest _not_ to use common words for names so fossify was the only possible choice...

**Reply by nehemiagurl** [reply]
that or "fossy" that was also mentioned in this thread.

**Comment by dekipantelija** [comment]
I guess foss apps is an acceptable middle ground. I'll ask brodie to make another video and lobby my choice up :P

**Reply by ciampix** [reply]
you mean fossy...

**Reply by Floppy-De** [reply]
thats my point too. If I do a search, I want all apps to be listed, and none out of this bundle of apps. So please no generic. And please be aware of autotranslated content. Thats no problem with a marky proper noun.

**Comment by ghost** [comment]
I really wish we could choose multiple names, or Ranked Vote, I'm fine with either Foss or Fossify, though I'd have to pick Fossify first because it's (as far as I know) unique, making the apps easy to find if and when some is like "Yeah I recommend Fossify Gallery." Much less likely to find unrelated Apps or websites, now and likely in the future.

**Reply by ghost** [reply]
Something I feel like I should add (Partly because I just had to deal with it) Is if we're too generic, it can be a pain for users to find anything. I recently had to try and figure out how to do something with Element, and I could barely find anything related to the Matrix Client, let along related to the issue I was looking into. I would prefer if we didn't have that same issue.

**Comment by guidow** [comment]
I don't like that you made a second poll here, especially that part where you claim the first one was inconclusive, when there was a very clear plurality for the "Pure _____" naming schema. I have no way of knowing what your true motivation was here, but I can tell you what it looks like to me as an outsider: It looks like you just didn't like the result of the first vote, and so decided to open a second poll in the hopes that people who voted in the first one won't notice in time.

Look, nobody is forcing you to decide on the name in a democratic way. It is your fork, you can name those apps however you want. But if you do offer that option, it's a really bad look to immediately proceed with something that at least has the looks of some very blatant vote manipulation.

This feels like the "Boaty McBoatface" thing all over again.

**Reply by naveensingh** [reply]
@guidow I haven't seen you in other comments so I'll assume you have been passively observing and not actively participating in the discussions. Please go through the replies [here](https://github.com/orgs/FossifyOrg/discussions/23#discussioncomment-7771190), [here](https://github.com/orgs/FossifyOrg/discussions/23#discussioncomment-7778537) and [here](https://github.com/orgs/FossifyOrg/discussions/23#discussioncomment-7790663).

> there was a very clear plurality for the "Pure _____" naming schema

You are not wrong about the plurality. [Pure currently has 44% votes](https://github.com/orgs/FossifyOrg/discussions/23) and that falls short of a simple majority (50% or more). I don't think 44% votes can be considered conclusive when we are trying to ensure a broader consensus. Almost 600 people chose something else.

**Reply by developist** [reply]
Now that you'd mention Boaty McBoatface, Fossy McFossFace is not a bad idea 😝

**Reply by mosiser** [reply]
What is the problem? If the "majority" voted for Pure in the first poll, they can vote again for Pure in this poll. They will win again.
Or are you afraid that, maybe, these voters will not come back, because they don't follow this project, or because they are not sent by someone else? 😉
In that case, the vote would be more "democratic"...

**Reply by ikanakova** [reply]
I think it's because often people vote first and then read the discussion and find out that Pure is such a common name that every other app has and would like to change their mind. `<irony>`It's really terrible that they now have the opportunity...`</irony>`

**Reply by dekipantelija** [reply]
> What is the problem? If the "majority" voted for Pure in the first poll, they can vote again for Pure in this poll. They will win again. Or are you afraid that, maybe, these voters will not come back, because they don't follow this project, or because they are not sent by someone else? 😉 In that case, the vote would be more "democratic"...

The problem is this is a classic move to tinker with votes. There is a larger representation, so those established glfeel left out. As a result, they call for a second vote, leaving many who voted behind because they are less involved (not less interested!) and then the established get their strength back. Naveen decided to make a poll, and did not close the vote to a certain criteria of voters. Newcomers have equally the right and playing with their involvement is simply antidemocratic.
IF the reason was a sub 50%, then you'd have chosen between onoy two. This here loooks like a re-roll. But whatever, it's not a vital topic, I'm just relieved the militants in this thread and topic aren't actively involved in politics

**Reply by mosiser** [reply]
> The problem is this is a classic move to tinker with votes. There is a larger representation, so those established glfeel left out. As a result, they call for a second vote, leaving many who voted behind because they are less involved (not less interested!) and then the established get their strength back. Naveen decided to make a poll, and did not close the vote to a certain criteria of voters. Newcomers have equally the right and playing with their involvement is simply antidemocratic.

Playing with votes?
Maybe, playing with votes is when an influential person can call his followers to vote for his personal preference.
This in when a poll become "antidemocratic": the results are in the hands of few powerful people.
The "established", as you call them, didn't see that coming, and now they are trying to fix the problem with a second poll.

I have nothing against "Pure". It's also my second choice. But it have to win with a majority of preferences, not with driven votes of people that come here only to please someone else...
People that are not interested and, maybe, they don't even notice this second poll.

Look at the numbers, 1/10 of the original poll. Ask yourself why...

**Reply by dekipantelija** [reply]
@mosiser that is the entire point I'm making. A tenth of votes because people who aren't as engaged end up not being informed about the second poll. Claiming its Brodies fault that Fossify isn't chosen is disingenuous. He telked about this fork, said he didn't like the name and that's it. The people who came didn't say they listened to Brodie, they agreed the name wasnt good.
Imaginr applying that to a public vote: "usually the voting population is ten times smaller, therefore this time it's not okay, because someone called in more people to vote" my first choice didn't pass, fine. But claiming the naming got hijacked is not ok.

**Reply by mosiser** [reply]
> @mosiser that is the entire point I'm making. A tenth of votes because people who aren't as engaged end up not being informed about the second poll. 

Who has to decide the names?
Naveen could have choosen the name by himself, but he decide to let people interested in the project to vote.
If you don't know about the second poll, probably you are not interested in following this project.
So, probably, 9/10 of the voters of the first poll doesn't really care about the name of the apps. Maybe, they were just killing some time...

> Claiming its Brodies fault that Fossify isn't chosen is disingenuous. He telked about this fork, said he didn't like the name and that's it. The people who came didn't say they listened to Brodie, they agreed the name wasnt good. 

No, he explicitly asked people to come here and don't let "Fossify" win. He said "Pure" would probably win.
Guess what? After a little time, votes for "Pure" increase vertically. Don't you see a connection?

> Imaginr applying that to a public vote: "usually the voting population is ten times smaller, therefore this time it's not okay, because someone called in more people to vote" my first choice didn't pass, fine. But claiming the naming got hijacked is not ok.

It's not about the number of people, it's about the quality of the votes.
Yes, even in political elections, sometimes votes may be considered invalid...
In this case they are not even considered invalid. Perhaps they are valid, a second poll will verify this.

**Reply by dekipantelija** [reply]
@mosiser "It's not about the number of people, it's about the quality of the votes.
Yes, even in political elections, sometimes votes may be considered invalid..."
Reminds me of some current days stories lol.
In Switzerland we made a referendum against a new law project to imcrease the price of highway tolls. We won, and therefore the government, to get the extra money, decided to increase the gas taxes and use that money for the highway. Asking after the thought to do a second round is at a similar level. There was an explicit answer given, and no other criteria for voting than having access to the vote. No fake account, no dead people voting. If the youtuber said pure would probably win, that's because it was already im second place before hisvideo

**Comment by developist** [comment]
Seem like Naveen already made up his/her mind with Fossify so lets just move on with the project already. Beside anyone can forked the original and naming it themselves lol.

**Reply by naveensingh** [reply]
> Seem like Naveen already made up his/her mind with Fossify

@developist I haven't even voted yet. And it's he/him/his.

**Reply by naveensingh** [reply]
> lets just move on with the project already

@developist the names aren't needed yet.

**Comment by dante-e-v** [comment]
What about using something like "Basic?" 

Examples: Basic Gallery, Basic Music Player, Basic Calendar, etc.

That or "Easy".

I know there's some that say let's do Fossy which I do think is better than Fossify but that sounds more like a group name like "The Fossy Group" or "The Fossify Group". Probably fine if that's the group name for continuing development of these apps.

**Reply by naveensingh** [reply]
Too late for new names, we'll pick [whatever wins.](https://github.com/orgs/FossifyOrg/discussions/23#discussioncomment-7790663) 

Besides, the apps are not basic anymore (especially the popular ones). I like Easy.

**Comment by erebion** [comment]
For how long will the poll be open? Some users want to change their choice depeding on what is leading.

**Reply by naveensingh** [reply]
As mentioned in the original post, we'll _maybe_ do a final round with the two most voted options. Probably after 1-2 days. There are no strict time constraints.

**Reply by dekipantelija** [reply]
@naveensingh i respect your commitment to letting everyone decide. I'd advocate that we keep whoever wins not, these poll things will be never-ending if we keep grouping, selecting 2, debate, and then vote again. Thanks again for the fork!

**Reply by DiagonalArg** [reply]
I think a top-two poll is probably the best, and probably should have been the second poll, but it is what it is.  As they're already working on the project, and don't need a name immediately, it doesn't appear there's a rush.

**Comment by kaoneko** [comment]
A few less generic suggestions (because there's not much variety in the poll):

**Naveen**

> Naveen is an Indian male given name and surname. The word means "new", "fresh", "young", "bright", "creative". Naveen is chiefly used in Indian languages, and its origin is Sanskrit.

_Naveen Calendar
Naveen Gallery
Naveen Notes_

Quite distinctive.

**Plesim**

An anagram of Simple.

_Plesim Calendar
Plesim Gallery
Plesim Notes_

Quite pleasant.

**Not For Sale**

_NFS Calendar
NFS Gallery
NFS Notes_

Quite professional.

**Reply by gitch22** [reply]
Love the "Naveen Calendar", etc idea. Ingenious and meaningful.

**Reply by ciampix** [reply]
I love them all...

**Comment by alekksander** [comment]
at this point i'm worried about the code more than the looks/name of this suite. not much going on since forked. simple mobile tools was to alternate default android applicaitons, but is this the case anymore, since there won't be dedicated android device in production? this gives opportunity to drop some of the less unique applications like clock, music player, launcher, voice recorder or flashlight.

**Comment by SRC267** [comment]
"Pure" is best for the general public. It's easy to search and easy to understand for the less technical user base, so you'll get more eyes on it. The others like fossy etc, will generally appeal to the more tech savvy.

**Reply by ciampix** [reply]
No. They won't get any public simply because of the too much used name, they will never be able to find it on the internet... bwt "pure calendar" seems to be so common that it appears to be already taken...

**Comment by inspector71** [comment]
With all due respect, the people who have never heard of GitHub or FOSS, and don't need to in order to install and try/use a decent, simplistic app, will get confused by any name with the lettering "foss" - in any variation. This survey is not representative of those potential users and thus should be taken with a grain of salt, to say the least.

The original apps are simple and thus powerfully easy to choose/ try without any cognitive dissonance for users.

Please don't destroy that.

**Reply by dekipantelija** [reply]
Why the thumbs down guys?

**Reply by ikanakova** [reply]
>Why the thumbs down guys?

Because I completely disagree with @inspector71 opinion (and it doesn't matter how many times he writes it here). I think the Foss/Fossify/Fossy name doesn't destroy anything. For example, I think that in the case of LibreOffice/OpenOffice, the Libre/Open in the name wasn't really important for people to choose it as an alternative to MS 365, the Office in the name was important. So whether Gallery is called Pure Gallery / Simple Gallery or Fossify Gallery doesn't matter, as long as it's Gallery free and ad-free and it will say in the description that it's the successor to Simple Gallery.

**Reply by TSnatch72** [reply]
>  I think that in the case of LibreOffice/OpenOffice, the Libre/Open in the name wasn't really important for people to choose it as an alternative to MS 365, the Office in the name was important. 

Extremely poor comparison. Name wasn't really important for LibreOffice and OpenOffice because 3 main things

1. OpenOffice is named Open, a word with a meaning that people understand, not FOSS
2. Both of them were (and LibreOffice still is) the default office suite of almost all linux distros
3. MS Office is a monopoly. MS Office is synonymous with office suites to the point people don't call them office suites but just office, word, excel, powerpoint, etc. people are either using it or looking for an alternative as good as it, and there are only a few alternatives.
4. When you google or search in youtube for alternatives to MS Office, LibreOffice is always there thanks to all the things above, which in turn makes it widely known for people looking for alternatives to ms office
5. Look how i said alternative. People don't look for office suites in general, people look specifically for MS Office alternatives. Except for maybe dialer and contacts, people are not looking for alternatives when they find SMT, they're looking for "best android gallery" "best android calendar" and so on. Let me say firsthand that most people installing dialer and contacts found them because of simple gallery, i doubt there's people actively looking to replace their phone's dialer and contacts.

Now, consider how we're not one of the most successful competitor to the de facto standard of the market which is highly talked about in  top videos and articles, but one of dozens of search results in a market so flooded as android and there's already a very similar equivalent which has a better, more descriptive name and it's already a top result on play store. Do you see how is not at all the case of LibreOffice and therefore it doesn't make sense to "imitate it" in something that is not a reason of its success? If anything, LibreOffice succeeded despite is name thanks to a ton of other factors and good decisions and then its success made the term "libre" more popular, not the other way around

**Comment by TSnatch72** [comment]
I think people that don't know what FOSS software is would get way more attracted and get way better the idea of a simple, debloated application much more by calling them something other than "Fossify" or "Foss". I think that by naming the applications like that we're just praying to the choire and damaging the project in the process. Simple Mobile Tools was so successful because it wasn't signaling hackers looking for FOSS but regular folk looking for simple apps without ads or paywalls. **"Pure"**, or **"Just"** work way better as a bridge to FOSS than Fossify would ever do. Remember, this fork is now going to compete with "**Simple** Gallery" and "**Simple** Calendar".

I have seen that there already existed an app called Pure gallery that's now delisted from playstore and there also is a "Pure calendar" in the AppStore, so i think that out of all of the options in the poll, "Just" is the best one. But my point here is that whatever the brand ends up being, i strongly suggest against either "fossify" or "foss". 

@Dante-E-Vergilson Suggested _**"Easy"**_ earlier, I really like that one too. Very similar meaning to "**Simple**", is an adjective people are looking for in apps, more probable for folk to search on the play store and haven't seen any application following that naming scheme there, only a gallery app that is called _**"EZ"**_ instead of _**"Easy"**_.

**Reply by ikanakova** [reply]
> Simple Mobile Tools was so successful because it wasn't signaling hackers looking for FOSS but regular folk looking for simple apps without ads or paywalls.

I'm not a hacker but a regular folk and I'm a fan of FOSS apps, that's strange isn't it?


I think you put an awful lot of importance on the name. In my experience, regular users choose an app based on features, PrintScreen, and the fact that it's free and ad-free.

**Reply by TSnatch72** [reply]
You are a fan of FOSS, that makes already less than 5% of the market. You're already not regular folk. Do you get it now? FOSS is a ridiculously tiny niche and by signaling a niche while you're trying to compete with something like SMT you're shooting yourself in the foot, that's not hard to understand.
"I'm not a hacker tho" you're fixating on the less important aspect of what i said

**Comment by inspector71** [comment]
If that app was titled FossyPrint, what would users think it does?

**Reply by inson1** [reply]
@inspector71 in that case it should be adfree calendar etc. right?

**Comment by inspector71** [comment]
Name is:

1. the very first identifier

2. the label that tells people what their tapping on, sharing to, amongst a
sea of icons and options

It's not everything but it's easily the highest priority.

People who say "you put too much cred into the name" are not addressing the
topic. Skirting the issue.

Name is not everything but it's a high priority. Particularly in this case
because it has to differentiate an well established product that will
initial do exactly the same as the 'rogue's version. If people only look at
the description of functionality and that's exactly the same as the
'Simple' tools, how does that help them decide?

**Comment by inspector71** [comment]
Libre

That's very short, very descriptive and although non-English, easily
understood to represent what the project is all about.

It's worked for projects in a similar situation.

**Reply by dekipantelija** [reply]
Sadly it's already taken by LibreOffice and I'm not sure bringing the confusion to with an already famous foss brand

**Reply by TSnatch72** [reply]
LibreOffice is not the only application using "Libre" as a prefix, so wouldn't consider "taken" because of it. Following that logic, Foss is already taken. I still wouldn't use Libre (or foss), though

**Comment by TheShanMan** [comment]
I'm techie and a big fan of FOSS and I agree that any form of "Foss" in the name is quirky to non-techies. I don't think this is the most important issue facing this suite, but it's good to give it due consideration as it's not ideal to change the name later if the original chosen name flops.

**Comment by inspector71** [comment]
Libre Calendar
Libre Gallery

Etc

That's my vote.

**Comment by bmslgpn** [comment]
I think that with 'Pure', the project will have to deal with conflicts over the names of existing commercial applications.
Personally, I would have preferred 'Just'.
But I think it's time to put an end to the vote, and even if I'm not a big fan of 'Fossify', this name has the merit of being absolutely unique and of materialising the DNA of the project.
It's just a name, after all, and the target users know exactly what they're installing. The quality of the applications is much more important than their name.

**Comment by jfdhuiz** [comment]
Use A/An

A Calendar
A Mail.
A Gallery.

Etc.

Or your identity. Whatever that is. Your nickname. Your realname. Your organization's name.

Either way, I don't care. I remove the Google ones and rename these apps in my launcher/shortcuts to remove the prefix. Because you don't have 5 Calendar and 8 Mail apps installed. At least, I don't.

There's quite a nice FOSS Calendar app out there called Etar. Which is apparently Arabic word for Calendar (???). That works too.

What I do care about is that you hurry up with providing these apps. I don't trust scum companies and scum devs who outsell like that.

**Comment by Brandonbr1** [comment]
Fossify. In my opinion, That's not a good name. It does not signify anything about how good the quality is.... Just because you all like Foss does not mean it is a good name. Same for libre, its not very good, does not signify anything that's good about it. No hate, these are my options being presented as to why this name is not good.

### Pure can refer to it being high quality and not full of ads, as an example
 Pure can refer to high quality or no ads, as someone here said. Whatever happens, happens. I just want a phone app that does not spy on me crazy! How there is only one. 

#### All other apps are kinda bad
I mean I tried emerald dailer and it does not have a call ui for, when you call a person.

## Here are my 2 cents on this situation;

It's *crazy* how hard the Dev dropped the ball on his suite of simple tools. Like wow. All he had to do was mention how it is unmaintained and someone could have picked it up like this or sell it off to another Foss Dev. He likely did not do his research or either he wanted money.

### [Small rant]
Idk either way I just hope this project releases on f-droid soon, so I can make calls, my imagine having to wait for this to release, just because there is no enought dialer apps. Your core function is to call, yet no-one else has managed to make an app like that, that is ok and does not steal your data.

**Reply by ciampix** [reply]
You wrote "All other apps are kinda bad", and (purposely?) forgot to mention that "all" means "proprietary". And you wrote that "foss" and "libre" are not good because they do not signify anything, and yet ... those terms signify something... guess what...

**Reply by dekipantelija** [reply]
@ciampix to be honest, there are so many bad foss apps as well. Open source doesn't equal quality

**Reply by ikanakova** [reply]
> @ciampix to be honest, there are so many bad foss apps as well. Open source doesn't equal quality

The beauty of FOSS, however, is that if you don't like something, you can customize it to your liking, so you can even fix a bad app to make it high quality. You can't do that with proprietary apps.

**Comment by keybreak** [comment]
How about **Smooth**? :sunglasses: 

P.S. I like **Pure** out of that list! :upside_down_face:

**Comment by ikanakova** [comment]
As the opponents of the name FOSS / Fossify / Fossy write here that the term FOSS is unfamiliar to the common user, the advantage of choosing name FOSS / Fossify / Fossy is that at least common users will become familiar with it. It will be such an advertisement of FOSS :-)

Edit: In my opinion, the most important thing is to finally choose a name (probably Fossify or Pure) so that the app can be distributed.

**Reply by bmslgpn** [reply]
That's exactly what I thought. You have to look on the bright side.

**Reply by TSnatch72** [reply]
No, that's now how it works. You don't use what you're trying to get people into as the bait, it's the other way around. You attract regular folk with something else they're familiar with and talks to them, then once they're in you present FOSS to them. Naming it "FOSS" won't make people unfamiliar with FOSS interested to it because you're placing FOSS as the very first filter. People unfamiliar with it won't click. Most of the people seeing the app will simply not install it because the name doesn't say anything to them, unlike, for example "Simple", which pops on the first results between heavily marketed apps with paid spots.

I'm sorry, but you simply don't understand how marketing works.

**Reply by ikanakova** [reply]
> No, that's now how it works. You don't use what you're trying to get people into as the bait, it's the other way around. You attract regular folk with something else they're familiar with and talks to them, then once they're in you present FOSS to them. Naming it "FOSS" won't make people unfamiliar with FOSS interested to it because you're placing FOSS as the very first filter. People unfamiliar with it won't click. Most of the people seeing the app will simply not install it because the name doesn't say anything to them, unlike, for example "Simple", which pops on the first results between heavily marketed apps with paid spots.

> I'm sorry, but you simply don't understand how marketing works.

But we're not trying to sell an app - this is FOSS, even if it will not called that. Plus, these apps will have the marketing benefit of getting the word out that they are the successor to Simple apps and that's why people will download them, not because they are called by the name e.g. Pure

**Comment by naveensingh** [comment]
I appreciate all the suggestions here and I like many of them but to avoid going down the slippery slope, I'm going to wrap this up with a final poll including Pure and Fossify: https://github.com/orgs/FossifyOrg/discussions/54

**To those who are concerned about discoverability:**
If the winning app name becomes a bottleneck, I'll consider changing it **later**. Given the nature of the project, I think any potential impact from an **app name** change later on won't be disastrous.

**Reply by bmslgpn** [reply]
Perhaps another solution for choosing the name would have been, instead of putting it to the vote, to ask everyone's opinion on each proposed name. Then, by collecting and analysing them, you could have made an informed and firm decision. As you are taking over the project, it is perfectly legitimate for you to choose a name that suits you first and foremost. If you develop a software suite whose name you don't like, you're going to lose motivation.

**Reply by naveensingh** [reply]
Yes, I am starting to think a poll might not have been the best idea for this, should've just stuck to good old brainstorming. Live and learn I guess.

**Reply by ikanakova** [reply]
> Yes, I am starting to think a poll might not have been the best idea for this, should've just stuck to good old brainstorming. Live and learn I guess.

I think there are more important things than a name and this experience will be valuable for those more important things in the future :-)
After all, it doesn't matter what the result of the final poll is, as @bmslgpn writes, it's important that you choose a name that suits you first and foremost.

**Reply by inson1** [reply]
I liked the pool tbh. 

Plus it gets more people to come here imo.

**Comment by inspector71** [comment]
Firefox was Phoenix (I think) and I admit, at the time they decided to change it, I went berserk thinking they'd have lost / confused so much of the attention Phoenix had generated.

Almost 2 decades later, Mozilla's Firenix / Pheofox (😲🤭) may have won and lost a lot of market share, been completely usurped by the big G, but clearly it survived a name and logo change quite easily 🤭😊🤔

I can admit when I'm wrong. It's not as painful as many high profile humans may fear 🙄😊

Thank you for your efforts to continue providing these valuable apps (tools) in the manner in which they were originally supplied.

**Comment by Ponytronics** [comment]
Instead of Fossify, what about Mossify, you know, like as a tribute to the character Moss in The IT Crowd?
"No, Jen. You don't need a subscription. It's FOSS, just like me but with a letter F!"

**Comment by Tourma** [comment]
I like Pure, but it might get conflated with PureOS or a number of other things.  Core I personally have some bad associations with.  Foss is used surprisingly little on F-Droid, (I feel it would be like using the word "Super" in SNES games,) but it still will likely produce multiple hits.  Fossify (or Fossy) is at least a unique set of characters.

**Comment by coxe87b** [comment]
Correct me if I'm wrong, but I'm assuming your developer organisation which will be the listed developer of the app suite is "FossifyOrg" and we are voting on the prefix name for all the apps.

+1 for Pure as there doesn't seem to be many other apps using that prefix that I could find on F-Droid or Aurora store. This prefix also seems to be the most popular solution so far, which brings us closer to a solution.

Some of the other suggestions are great as well but if we keep adding suggestions we will drown in choice and it will make it more likely to be a stale mate

**Reply by inson1** [reply]
@coxe87b voting already ended

**Comment by y1111** [comment]
i see this already ended . but i really would like add that actually all the names here dont really express what this project is , what the real differential it has . 

the real unique thing on this project is the theming ,the customization, no the foss part  . of course being foss is important but theres many others foss apps for every single app here . the real unique thing is to customize the themes and be able to apply to all the apps . this is the thing here , thats what is not found somewhere else and with so many apps .

maybe a name involving custom, themes , customized apps , yourthemeapps , something like in that idea would make more sense.

i even just read in other post the dev talking about the gallery pro having paid sdk's so it just already show the this name is not what really describes better the project

**Reply by inson1** [reply]
dev in smt, it was deleted in fossify release

**Reply by ikanakova** [reply]
> i even just read in other post the dev talking about the gallery pro having paid sdk's so it just already show the this name is not what really describes better the project

You mean this [discussion](https://github.com/FossifyOrg/Gallery/discussions/6?sort=new#discussioncomment-7928256), where it is clearly stated that the proprietary sdk added to the application by Tibor, who apparently always ignored the GNU license (and the whole FOSS principle), has already been removed from the application?
